src/less/floating-label/floating-label.less

```diff
@@ -106,3 +106,13 @@ label.floating-label__label--invalid {
         transform-origin: right;
     }
 }
+
+// Autofill
+
+label.floating-label__label:has(+ .textbox :autofill) {
```

Comment:
The `transform` is getting overwritten. The selector seems to need more specificity...
<img width="374" alt="image" src="https://github.com/eBay/skin/assets/1675667/bd108000-b05c-4fc6-b5c3-05086f4287ce">

Reply:
...or possibly, that transform is from a different selector and the `:has` isn't getting applied at all?